modules/http/src/main/scala/ch/epfl/bluebrain/nexus/service/http/routes/StaticResourceRoutes.scala

```diff
@@ -0,0 +1,65 @@
+package ch.epfl.bluebrain.nexus.service.http.routes
+
+import java.io.File
+import java.util.regex.Pattern.quote
+
+import akka.http.scaladsl.model.{StatusCodes, Uri}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import ch.epfl.bluebrain.nexus.commons.http.JsonLdCirceSupport._
+import io.circe.Json
+import io.circe.parser.parse
+
+import scala.io.Source
+
+class StaticResourceRoutes(resourcePath: String, prefix: String, baseUri: Uri) {
+
+  private def contentOf(file: File): String =
+    Source.fromFile(file).mkString
+
+  private def contentOf(file: File, replacements: Map[String, String]): String =
+    replacements.foldLeft(contentOf(file)) {
+      case (value, (regex, replacement)) => value.replaceAll(regex, replacement)
+    }
+
+  private val baseReplacement: Map[String, String] = Map(quote("{{base}}") -> baseUri.toString)
+
+  private def folderContents(folder: File): Map[String, Json] = {
+    folder
+      .listFiles()
+      .toList
+      .filter(f => f.isFile && f.getName.endsWith(".json"))
+      .flatMap { file =>
+        parse(contentOf(file, baseReplacement)).toOption.map { json =>
+          file.getName.stripSuffix(".json") -> json
+      }
+    } toMap
```

Comment:
I think we shouldn't abuse infix notation, much less postfix one.
https://docs.scala-lang.org/style/method-invocation.html#postfix-notation

Reply:
+1 on this one